import "imports_submessage.proto";

message ParentMessageImports {
  optional SubMessageImports c = 3;
}
